src/Avalonia.Base/Media/Fonts/FontCollectionBase.cs

```diff
@@ -348,7 +348,7 @@ internal static bool TryFindWeightFallback(
             return false;
         }
 
-        internal static Typeface GetImplicitTypeface(Typeface typeface, out string normalizedFamilyName)
+        public static Typeface GetImplicitTypeface(Typeface typeface, out string normalizedFamilyName)
```

Comment:
It is not very clear from "Implicit" what this API does. Why is it not an instance method on Typeface?